optking/loggingconfig.py

```diff
@@ -8,9 +8,12 @@
         "time_severity_message": {
             "format": "%(asctime)s - %(levelname) - %(message)s"
             },
-        "severity_message": {
-            "format": "[%(asctime)s] [%(levelname)s] [%(name)s]: %(message)s"
+        "severity_name_message": {
+            "format": "[%(levelname)s] [%(name)s]: %(message)s"
```

Comment:
may want to check out the python "f-string". this would become `"format": f"[{levelname}] [{name}]: {message}"`

Reply:
Apparently, f strings don't work for the creation of a logging Formatter due to the logging modules internal use of %-formatting. `%(asctime)s`, `{asctime}`, and `$asctime` are all acceptable but `f"{asctime}"` is not.

*shrugs*